themis/spec/secure-cell/examples.md

```diff
@@ -0,0 +1,190 @@
+---
+weight: 40
+title:  Secure Cell examples
+---
+
+# Secure Cell examples
+
+Now let's look at and dissect sample data protected by Secure Cell.
+You can try it out yourself using [command-line utilities](/themis/debugging/cli-utilities/) if you have Themis installed,
+or with [Themis Server](/themis/debugging/themis-server/) from your web browser.
+
+## Symmetric key encryption
+
+With the following inputs
+for [the `scell_seal_string_echo` utility](/themis/debugging/cli-utilities/):
+
+| Input          | Value |
+| -------------- | ----- |
+| encryption key | `au6aimoa8Pee8wahxi4Aique6eaxai2a` |
+| plaintext      | `encrypted message` |
+| context data   | `additional context` |
+
+Secure Cell in Seal mode produces the following output (encoded in base64):
+
+    AAEBQAwAAAAQAAAAEQAAAM5da3KkReYC7++OPbrI13UycoVi3s01Ji64WQ/KIe+3oF8cgLle19WC+tnaCg==
```

Comment:
does it look fine without being surrounded by ```?

Reply:
> does it look fine without being surrounded by ```?

Indentation is a different way to introduce preformatted blocks of text in Markdown. It makes raw text a bit easier to read, so I opt for it from time to time for non-code blocks. And yes, it renders fine.